src/components/reviews/characteristicsRadio.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { MEANINGS } from "../../characteristicMeanings.js";
+
+class CharacteristicsRadio extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+
+    };
+
+  }
+
+
+
+
+  render() {
+    let { characteristic } = this.props;
```

Comment:
Good use of object de-structuring here. Really makes the use of props understandable.